PSKNodeServer/PSKNodeServer/API/scanditScan/ScanditScannerViewController.swift

```diff
@@ -6,22 +6,78 @@
 //
 
 import UIKit
+import ScanditBarcodeCapture
 
 class ScanditScannerViewController: UIViewController {
     @IBOutlet weak var apiKeyLabel: UILabel!
     private let scanditApiKey: String
+    private let symbologies: [Symbology]
+    private let dataCaptureContext: DataCaptureContext
+    private var barcodeCapture: BarcodeCapture
     
     override func viewDidLoad() {
         super.viewDidLoad()
         apiKeyLabel.text = scanditApiKey
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        barcodeCapture.addListener(self)
+        
+        let cameraSettings = BarcodeCapture.recommendedCameraSettings
+        Camera.default?.apply(cameraSettings)
+        dataCaptureContext.setFrameSource(Camera.default)
+        Camera.default?.switch(toDesiredState: .on)
+        
+        let captureView = DataCaptureView(context: dataCaptureContext, frame: view.bounds)
+        captureView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(captureView)
+        
+        _ = BarcodeCaptureOverlay(barcodeCapture: barcodeCapture, view: captureView)
+    }
+    
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    init(with scanditApiKey: String){
+    init(with scanditApiKey: String, andWith symbologies: [Symbology]){
         self.scanditApiKey = scanditApiKey
+        self.symbologies = symbologies
+        self.dataCaptureContext = DataCaptureContext(licenseKey: scanditApiKey)
```

Comment:
If `scanditApiKey` is used only here then you don't need to store it as instance constant

Reply:
It is used on screen as label, but it was only temp for dev,I'll remove it together with the label